### DISCUSSION TITLE
Advantage over sinchiro's builds?

### DISCUSSION THREAD

**Discussion by wyup** [zhongfly/mpv-build#107] [General]
Hi, thank you for mpv builds.
As a enthusiast, I would like to know the advantage of your builds over sinchiro's.
It seems it is based on the same libraries. Does it add more updated or additional ones? Better toolchain compiler and binary performance?

**Comment by zhongfly** [comment]
Same as sinchiro.
No difference except I release more frequently.

**Reply by zhongfly** [reply]
If I had to say one advantage, it would probably be that compared to sinchiro, I would try to ensure released every day.

**Reply by wyup** [reply]
you don't do a gcc build like shinchiro? Any advantage of this particular build for intel/AMD?
[mpv-x86_64-gcc-20240513-git-61f72bd.7z](https://github.com/shinchiro/mpv-winbuild-cmake/releases/download/20240513/mpv-x86_64-gcc-20240513-git-61f72bd.7z)

**Comment by Obegg** [comment]
Well, now this repo has two advantages over `shinchiro`:
1. https://github.com/zhongfly/mpv-packaging/commit/3bcec15e9d5085350c8bc169eaa0d6612ade7981 - `updater.bat` can now download `yt-dlp` or `youtube-dl` by changing `settings.xml`, which means the user can download mpv unattended now by using the following `settings.xml`:
```xml
<settings>
  <channel>daily</channel>
  <arch>x86_64-v3</arch>
  <autodelete>true</autodelete>
  <getffmpeg>true</getffmpeg>
  <getytdl>ytdlp</getytdl>
  <ytdlpchannel>master</ytdlpchannel>
</settings>
```
2. https://github.com/zhongfly/mpv-packaging/commit/8ee448ece4d92a30596a92ef97730913da8f285d - `updater.bat` can now download `stable`, `nightly` and `master` update releases for `yt-dlp` and also adds it to `settings.xml` (as can be seen in the above).

@zhongfly We should probably open a PR on `shinchiro` so our changes sync up?

**Reply by zhongfly** [reply]
https://github.com/shinchiro/mpv-packaging/pull/55
Next time if you have any new ideas, I suggest you open PR on shinchiro first.
Sorry, I just don't want to spend too much time on the mpv-packaging repository.

**Reply by Obegg** [reply]
Thank you, I tried to somehow do it myself for a few minutes and some googling but did not figure it out yet.
I did it here because I've seen his repo has unmerged PRs, without any reply so I was thinking it was inactive, I still think it's inactive, so we'll see how long it takes to merge this PR.
But yes, I do appreciate your time and some fixes and reviews, I don't think I will change anything else now, that's all I wanted to change.
